lib/erlef/data/query/academic_paper.ex

```diff
@@ -0,0 +1,121 @@
+defmodule Erlef.Data.Query.AcademicPaper do
```

Comment:
Normally I default to a context module of sorts, so Erlef.AcademicPapers, but I don't have strong feelings about it in this context.

Reply:
I like the separation. Specifically, isolating the query & schema modules then, if necessary, having a context that utilizes the query module, or maybe multiple query modules is my common pattern.

Reply:
It's a pattern I'm familiar with 😁 I don't have any strong feelings on this though.